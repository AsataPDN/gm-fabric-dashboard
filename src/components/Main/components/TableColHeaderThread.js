import TableColHeader from "./TableColHeader";

const TableColHeaderThread = TableColHeader.extend`
  flex: 0 1 6%;
  display: flex;
  align-items: center;
`;
export default TableColHeaderThread;
