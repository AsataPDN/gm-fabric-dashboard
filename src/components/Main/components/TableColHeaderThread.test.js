import React from "react";
import { shallow } from "enzyme";
import TableColHeaderThread from "./TableColHeaderThread";

describe("TableColHeaderThread", () => {
  it("matches snapshot", () => {
    const aTableColHeaderThread = shallow(<TableColHeaderThread />);
    expect(aTableColHeaderThread).toMatchSnapshot();
  });
});
