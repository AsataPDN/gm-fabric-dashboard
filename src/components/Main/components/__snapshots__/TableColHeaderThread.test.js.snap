// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableColHeaderThread matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 12%;
  -ms-flex: 1 1 12%;
  flex: 1 1 12%;
  text-align: left;
  padding-right: 16px;
  padding-top: 7px;
  padding-bottom: 7px;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  -webkit-flex: 0 1 5em;
  -ms-flex: 0 1 5em;
  flex: 0 1 5em;
}

.c0:first-child {
  padding-left: 16px;
}

<div
  className="c0"
/>
`;
